onboardingtaskapp.client/src/components/Customer/Edit.jsx

```diff
@@ -1,23 +1,39 @@
 import { React, useState, useEffect } from 'react';
-import { Modal, Button } from 'semantic-ui-react';
+import { Modal, Button, Message } from 'semantic-ui-react';
 
-export default function Edit({ item }) {
+export default function Edit({ item, isUpdated }) {
 
+    // states
     const [open, setOpen] = useState(false);
-    const handleOpen = () => setOpen(true);
-    const handleClose = () => setOpen(false);
-
+    const [message, setMessage] = useState(null); 
 
     const [formData, setFormData] = useState({        
         name: '',
         address: '',
     });
-
+      
     useEffect(() => {
-        
-        const fetchCustomer = async () => {
+
+        if (!item.customer.id) {
+            throw new Error('Invalid Id');
+        }
+                
+        fetchCustomer(item);
+
+    }, [item.customer.id]);
+
+    // const
+    const API_END_POINT = `https://onboardsite.azurewebsites.net/api/Customer/`;
```

Comment:
This is actually good and you can set this globally. Instead of initialising it here you can actually create a separate folder called **services** and then an **environment** js file and you can set something like:

export const API_END_POINT = `https://onboardsite.azurewebsites.net/api/`;
export const CustomerPath = `Customer/`;
etc..

and then you can import these in your files